pkg/utils/utils.go

```diff
@@ -13,13 +13,6 @@ import (
 	"github.com/confluentinc/cli/v3/pkg/errors"
 )
 
-func Max(x, y int64) int64 {
-	if x > y {
-		return x
-	}
-	return y
-}
-
 func TestEq(a, b []string) bool {
 	// If one is nil, the other must also be nil.
 	if (a == nil) != (b == nil) {
```

Comment:
`slices` has a `Equal` function now, so we can remove this.

Reply:
Nice find!